angular/core/services/intercom_service.coffee

```diff
@@ -5,7 +5,7 @@ angular.module('loomioApp').factory 'IntercomService', ($rootScope, $window, App
     key: group.key
     name: group.name
     description: group.description.substring(0, 250)
-    admin_link: AppConfig.baseUrl+"/admin/groups/#{group.key}"
+    admin_link: AppConfig.baseUrl+"admin/groups/#{group.key}"
```

Comment:
Looking at angular_helper.. it's not so easy. I wonder if we need a compose url method that removes double-ups. uh.. why is this hard, @gdpelican what do you think?